docs/sni-limitation.rst

```diff
@@ -0,0 +1,35 @@
+.. _sni-limitation:
+
+Configure SNI TLS Load Balancer
+===============================
+
+Background
+----------
+
+Creating a new *TERMINATED_HTTPS* load balancer(LB for short) with new certificate(s) on OpenStack dashboard will trigger a new virtual server(VS for short) with new SSL profile(s) created on BIGIP side.
```

Comment:
Don't abbreviate load balancer and virtual server. If you don't want to have to type them out every time, create substitutions for them in the conf.py file (e.g., `|lb|` and `|vs|`).

```rst
When you create a new load balancer on the |dashboard-long| that uses *TERMINATED_HTTPS* and a new certificate(s), the F5 Agent creates a new virtual server and SSL profile on your BIG-IP device.
```

Reply:
done